frontend/src/pages/SubProjects/actions.js

```diff
@@ -101,9 +103,19 @@ export function setProjectHistoryOffset(offset) {
   };
 }
 
-export function fetchProjectHistory(projectId, offset, limit, showLoading = false) {
+export function fetchFirstProjectHistory(projectId, offset, limit, showLoading = false) {
   return {
-    type: FETCH_PROJECT_HISTORY,
+    type: FETCH_FIRST_PROJECT_HISTORY,
+    projectId,
+    offset,
+    limit,
+    showLoading
+  };
+}
+
+export function fetchNextProjectHistory(projectId, offset, limit, showLoading = false) {
```

Comment:
Please rename e.g. `fetchProjectHistoryItems(projectId, page)` and name the arguments according to the business domain. With this signature we could event drop the call to fetch the most-recent history